src/test/mochitest/browser_dbg-console.js

```diff
@@ -41,6 +41,6 @@ add_task(async function() {
   pressKey(dbg, "Escape");
 
   // Second time to hide console
-  pressKey(dbg, "Escape");
+  pressKey(dbg, "CtrlEscape");
```

Comment:
Adding this key code worked but it feels wrong. 🙁 @jasonLaster Care to dig in a bit?